cl/alerts/tasks.py

```diff
@@ -62,20 +89,54 @@ def send_docket_alert(d_pk, since):
 
+        trigger = False
         if new_des.count() > 0:
+            trigger = True
+        elif (
+            settings.PACER_USERNAME
+        ):  # if no credentials available, scraping is pointless. Required to pass tests.
+            # No new docket entries, check date_last_filing
+            if (
+                not docket.date_last_filing
+                or docket.date_last_filing < datetime.today().date()
+                and docket.date_last_filing < since.date()
+            ):  # Scrape iquery if date_last_filing is before today and before the last trigger
+                update_docket_info_iqeury(docket)
+            if docket.date_last_filing > since.date():
+                trigger = True
+        if trigger:
             # Notify every user that's subscribed to this alert.
-            case_name = trunc(best_case_name(docket), 100, ellipsis="...")
-            subject_template = loader.get_template("docket_alert_subject.txt")
-            subject = subject_template.render(
-                {
-                    "docket": docket,
-                    "count": new_des.count(),
-                    "case_name": case_name,
-                }
-            ).strip()  # Remove newlines that editors can insist on adding.
-            email_context = {"new_des": new_des, "docket": docket}
-            txt_template = loader.get_template("docket_alert_email.txt")
-            html_template = loader.get_template("docket_alert_email.html")
+            if new_des.count() > 0:
+                case_name = trunc(best_case_name(docket), 100, ellipsis="...")
+                subject_template = loader.get_template(
+                    "docket_alert_subject.txt"
+                )
+                subject = subject_template.render(
+                    {
+                        "docket": docket,
+                        "count": new_des.count(),
+                        "case_name": case_name,
+                    }
+                ).strip()  # Remove newlines that editors can insist on adding.
+                email_context = {"new_des": new_des, "docket": docket}
+                txt_template = loader.get_template("docket_alert_email.txt")
+                html_template = loader.get_template("docket_alert_email.html")
+            else:
+                case_name = trunc(best_case_name(docket), 100, ellipsis="...")
+                subject_template = loader.get_template(
+                    "pacer_docket_alert_subject.txt"
+                )
+                subject = subject_template.render(
+                    {"docket": docket, "case_name": case_name,}
+                ).strip()  # Remove newlines that editors can insist on adding.
+                email_context = {"docket": docket}
+                txt_template = loader.get_template(
+                    "pacer_docket_alert_email.txt"
+                )
+                html_template = loader.get_template(
+                    "pacer_docket_alert_email.html"
+                )
```

Comment:
One other thing: We should remove the duplication in this code. I think the case_name and the subject line rendering can move out of the if/else?

Reply:
Yeah, I guess the render can move out, we just won't use the count variable in the template.